docs/install.md

```diff
@@ -0,0 +1,21 @@
+# Installation
+
+Download the last PHAR release:
+```
+wget http://inviqa.github.io/dock-cli/downloads/dock-cli-latest.phar -O dock-cli
+chmod +x ./dock-cli
+sudo mv dock-cli /usr/bin/dock-cli
+```
+
+## Install Docker on your system
+
+To install the Docker development environment run:
+
+```
+dock-cli docker:install
+```
+
+**Note:** This sets environment variables which are loaded for the current shell and any opened after running it.
```

Comment:
Because it's out of the scope of the PR I'll merge, but I'm happy to discuss (and probably merge) a PR that change this behaviour @mareg :wink: